qiskit/ignis/verification/entanglement/analysis.py

```diff
@@ -92,7 +120,13 @@ def sin_plotter(self, x, y, y_m=None):
             plt.xlabel(r'\phi')
             plt.ylabel('Normalized Contrast')
 
-    def get_fourier_info(self, qn, x, y, y_m, p_dict):
+    def get_fourier_info(self,
+                         qn: int,
+                         x: np.array,
+                         y: np.array,
+                         y_m: np.array,
+                         p_dict: Dict
+                         ) -> Dict:
         """
         Get fourier trans. data/plot of both mqc and po exps.
```

Comment:
There's no `Args` sections for these. You've added types but no description of how they're used

Reply:
Done, thanks.